Jenkinsfile

```diff
@@ -160,7 +160,8 @@ helpers.rootLinuxNode(env, {
                             println "Building Docker"
                             sh '''
                                 set +x
-                                docker build -t keybaseprivate/kbfsfuse --build-arg KEYBASE_TEST_ROOT_CERT_PEM=\"$(echo $KBFS_DOCKER_CERT_B64 | base64 -d)\" --build-arg KEYBASE_TEST_ROOT_CERT_PEM_B64=\"$KBFS_DOCKER_CERT_B64\" .
+                                KEYBASE_DOCKER_CERT="$(echo $KEYBASE_DOCKER_CERT_B64 | base64 -d)";
```

Comment:
Should this be `KBFS_DOCKER_CERT` and `KBFS_DOCKER_CERT_B64`? If so, what does it change?

Reply:
Ah right it's supposed to be `KBFS_DOCKER_CERT`. It doesn't change anything; just populates one more env var in the line below. But now I'm not so sure it's gonna work ... I'll let you know when it's ready. Thanks!